src/types.d.ts

```diff
@@ -1,2 +1,4 @@
 declare module '@identity.com/credential-commons/src';
+declare module '@identity.com/dsr';
 declare module 'crypto';
+declare module 'ramda';
```

Comment:
We will need to standardize this, please open a tech debt ticket to change the whole repo to ramda

Reply:
I migrated lodash to ramda